godtools/App/Features/SetupParallel/SetupParallelLanguageViewModel.swift

```diff
@@ -48,6 +48,8 @@ class SetupParallelLanguageViewModel: NSObject, SetupParallelLanguageViewModelTy
         
         super.init()
         
+        setupParallelLanguageAvailability.markSetupParallelLanguageViewed()
+        
         reloadData()
         
         setupBinding()
```

Comment:
I didn't catch this before, but when this viewModel is deallocated can you remove this observer. languageSettingsService.parallelLanguage